eng/Packages.Data.props

```diff
@@ -114,20 +114,29 @@
 
   <!-- Track 2 specific versions -->
   <ItemGroup Condition="'$(IsClientLibrary)' == 'true'">
-    <PackageReference Update="Azure.ClientSdk.Analyzers" Version="0.1.1-dev.20200116.2" />
+
+    <!-- BCL packages -->
     <PackageReference Update="System.Memory" Version="4.5.3" />
+    <PackageReference Update="System.Diagnostics.DiagnosticSource" Version="4.6.0" />
+    <PackageReference Update="System.Threading.Channels" Version="4.6.0" />
+    <PackageReference Update="System.Text.Json" Version="4.6.0" />
+
+    <!-- Build time packages -->
+    <PackageReference Update="Azure.ClientSdk.Analyzers" Version="0.1.1-dev.20200116.2" />
     <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0" />
+    <PackageReference Update="Microsoft.DotNet.GenAPI" Version="5.0.0-beta.19552.1" />
+    
+    <!-- Azure SDK packages -->
+    <PackageReference Update="Azure.Storage.Blobs" Version="12.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsExtensionClientLibrary)' == 'true'">
```

Comment:
Doesn't this increase the risk of dependencies getting out of date? I don't see any benefit here since the compiler will elide any references that aren't needed and, in fact, these don't actually add package references anyway - just update existing ones with a version.

Reply:
This condition is to prevent "normal" azure SDKs from taking dependency on "extensions" packages that are intended only for a few selected plugin packages.